ui/src/components/Drawer/index.vue

```diff
@@ -26,8 +26,8 @@ const connectionStore = useConnectionStore();
 
 const drawerTabs = [
   {
-    name: 'hotkeys',
-    label: t('Hotkeys'),
+    name: 'general',
+    label: t('General'),
     icon: KeyboardIcon,
     component: Keyboard,
   },
```

Comment:
There is no need for further checks since there appear to be no grammatical, lexical, or punctuation errors in the provided text. The code appears to be concise and clean according to best practices. The "const" variable declarations are placed correctly at the beginning of every function definition. There doesn't seem to be an issue with spelling or syntax errors. If you have other specific questions about this code or anything else needing assistance, please feel free to clarify!